manifest.json

```diff
@@ -33,5 +33,8 @@
     "quick-find-search-links": {
       "description": "Search all links"
     }
-  }
+  },
+  "permissions": [
```

Comment:
I had some confusing results. After a bit of tinkering I managed to get the permission prompt by making sure to do `runtime.sendMessage()` from within the event handler instead of after a timeout. What is the purpose of the timeout? I should know if I'm going to do surgery on it.

But the stranger issue is that Chrome and Opera seemed to completely ignore the permissions. They always permitted the new tab or window action, even when `chrome.permissions.request()` called back with `granted == false` or I omitted `"tabs"` from manifest.json. (For Chrome I did the latter before even loading the extension.) I can code as if they actually enforced the permissions, but it leaves me uncertain that I can trust what I see working on my machine.